### DISCUSSION TITLE
add mapping to clean search entry like we do in Linux shell: Ctrl-u

### DISCUSSION THREAD

**Discussion by voyeg3r** [nvim-telescope/telescope.nvim#1517] [Ideas]
Sometimes we start searching for a given string but we change our mind and erase that search and start typing something new, I already have this option or it would/should be added?

In the Linux shell, we type Ctrl-u and erase what we have typed and even can type Ctrl-y to paste what we have erased with Ctrl-u

**Comment by Conni2461** [comment]
Just unmap the current use of `<c-u>` (scrolling previewer up)

```lua
local actions = require('telescope.actions')
require('telescope').setup{
  defaults = {
    mappings = {
      i = {
        ["<c-u>"] = false
      },
    },
  }
}
```
